bloch_sphere/animate_bloch.py

```diff
@@ -419,8 +445,12 @@ def run_from_command_line():
         'Save an mp4 video instead of a GIF')
     parser.add_argument('--fps', type=float, default=20, help=
         'Sets the animation frame rate')
+    parser.add_argument('--style', type=str, choices=['sphere', 'arrows'], 
+        default='sphere', help='Different style of showing rotated bloch sphere. '
```

Comment:
Suggested help: `'The style to draw the Bloch sphere.  E.g. draw the whole sphere or just draw the axis arrows.'`

Reply:
Done.